recipes/doubtlab/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "doubtlab" %}
+{% set version = "0.1.1" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e2065c916820ac6008883032594c14a0a660862cb012f3fac152a97f4c3c64b4
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+    - cleanlab >=1.0
+    - pandas >=1.3.3    
+    - scikit-learn >=1.0.0
+
+test:
+  imports:
+    - {{ name|lower }}
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://koaning.github.io/doubtlab/
+  summary: Don't Blindly Trust Your Labels
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  description: |
+    This repository contains general tricks that may help you find bad, 
+    or noisy, labels in your dataset. The hope is that this repository 
+    makes it easier for folks to quickly check their own datasets before 
+    they invest too much time and compute on gridsearch.
+  doc_url: https://koaning.github.io/doubtlab/
+  dev_url: https://github.com/koaning/doubtlab
+
+extra:
+  recipe-maintainers:
+    - sugatoray
+    - koaning
```

Comment:
@koaning are you OK  being a maintainer here?

Reply:
https://github.com/koaning/doubtlab/issues/14#issuecomment-969213611

Reply:
@ocefpaf I asked @koaning about his preference, before adding him as another maintainer. He had approved of it (please see [here](https://github.com/koaning/doubtlab/issues/14)). Also, since he is the maintainer of the library `doubtlab`, this would mean both PyPI and conda-forge versions will be easier to keep updated.